tests/ui/conftest.py

```diff
@@ -48,19 +141,7 @@ def jwt_token(base_url, jwt_issuer, jwt_secret):
 
 
 @pytest.fixture
-def user(base_url, fxa_account, jwt_token):
-    user = {
-        'email': fxa_account.email,
-        'password': fxa_account.password,
-        'username': fxa_account.email.split('@')[0]}
-    url = '{base_url}/api/v3/accounts/super-create/'.format(base_url=base_url)
-    params = {
-        'email': user['email'],
-        'username': user['username'],
-        'password': user['password'],
-        'fxa_id': fxa_account.session.uid}
-    headers = {'Authorization': 'JWT {token}'.format(token=jwt_token)}
-    r = requests.post(url, data=params, headers=headers)
-    assert requests.codes.created == r.status_code
-    user.update(r.json())
-    return user
+def variables(tmpdir, variables):
+    with tmpdir.join('variables.json').open() as f:
+        variables.update(json.load(f))
+        return variables
```

Comment:
`create_superuser` should depend on the `variables` fixture, create the temporary file for the API credentials and add the values directly to the `variable` object. We shouldn't need to override this fixture or depend on two separate calls to `tmpdir`.